content/ja/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: OpenTelemetry
+description: >-
+  効果的な観測を可能にする、高品質でユビキタスかつポータブルなテレメトリー
+developer_note:
+  blocks/coverコラム（以下で使用）は、ファイル名に "background" を含む画像ファイルを背景画像として使用します。
+show_banner: true
+limit_banner: 2 # Default to 2, adjust as needed
+banners:
+  - message:
+      '<i class="fas fa-bullhorn"></i> Join us for [**OTel Community Day** on
+      June
+      25th](https://events.linuxfoundation.org/open-telemetry-community-day/)!'
+    to: 2024-06-25
+  - message:
+      '<i class="fas fa-bullhorn"></i> Help improve OTel and Prometheus
+      interoperability: [complete our
+      survey](https://forms.gle/bZAG9f7udoJsjZUG9) by May 31'
+    to: 2024-05-31
+---
+
+<div class="d-none"><a rel="me" href="https://fosstodon.org/@opentelemetry"></a></div>
+
+{{< blocks/cover image_anchor="top" height="max" color="primary" >}}
+
+<!-- prettier-ignore -->
+![OpenTelemetry](/img/logos/opentelemetry-horizontal-color.svg)
+{.otel-logo}
+
+<!-- prettier-ignore -->
+{{% param description %}}
+{.display-6}
+
+<div class="l-primary-buttons mt-5">
+
+- [より詳しく学ぶ](/docs/what-is-opentelemetry/)
+- [デモを試す](/docs/demo/)
+- [インテグレーションを探す](/ecosystem/integrations)
+
+</div>
+
+<div class="h3 mt-4">
+<a class="text-secondary" href="/docs/getting-started/">Get started</a> based on your role
```

Comment:
Wouldn't it be better to translate this line too?

Reply:
thank you for catching this!